### DISCUSSION TITLE
How to Stop gif?

### DISCUSSION THREAD

**Discussion by Devkjk** [d2phap/ImageGlass#1880] [Q&A]
Ctrl + space , it didn't stop.
Tell me How to stop and play.

**Comment by d2phap** [comment]
Can you try using the menu or the Frame Navigation tool?

![image](https://github.com/d2phap/ImageGlass/assets/3154213/abc3fb0a-d2b3-4738-9379-ad11ed8c4b56)

**Comment by Devkjk** [comment]
Thank you! I will try it.
